Readability.js

```diff
@@ -1689,6 +1695,26 @@ Readability.prototype = {
   },
 
   /**
+   * Clean out elements whose id/class combinations match specific string.
+   *
+   * @param Element
+   * @param RegExp match id/class combination.
+   * @return void
+   **/
+  _cleanMatchedNodes: function(e, regex) {
+    var next = this._getNextNode(e);
```

Comment:
(To be clear, this method was written so you can pass it `articleContent` without iterating over the children with `_forEachNode`. The end marker node is so that it doesn't look for anything after it's finished searching the `articleContent` node, though I'm not entirely sure if that's necessary... does `articleContent` have any parents or siblings at the point where this code runs?)